modules/voice/bot.py

```diff
@@ -67,9 +68,52 @@ async def join(self, channel, vc):
                     f"Connecting to channel: <{channel}> timed out."
                 )
 
-    async def search_and_play(self, ctx, search, count):
-        sources = await self.search(ctx, search, count)
-        return await self.play(sources)
+    async def search_and_play(self, ctx, search: str, count):
+        OPTIONS_COUNT = 5  # TODO: number can be added to settings
+        sources = await self.search(ctx, search, OPTIONS_COUNT if count == 1 else count)
```

Comment:
Ok what if I want get only 1 song?

Reply:
You if you set it to one you and you will send link to yt you will get one song.
If you will send search steing you will get an option to select from 5 search results.
If you ste this number to any other then 1 you will get that amount of songs (0 is all results / usefull for playlist).